### PR TITLE
aio: generate api list json

### PR DESCRIPTION
**Pull request by petebacondarwin**

### CODE REVIEW

**Review by gkalpak — CHANGES_REQUESTED** [review]
I wonder what should happen for params without an `=`. E.g. `x/y?z`. Currently, it would be `{z: undefined}` I think, but maybe `null` would make more sense.

In any case, there should be a test for this.
Other than that LGTM.

**Comment by petebacondarwin** [comment]
Updated and force pushed after @gkalpak's comments. PTAL

File: aio/src/app/embedded/api/api.service.ts [hunk 1]
```diff
@@ -90,16 +81,6 @@ export class ApiService implements OnDestroy {
...
-    function normalizeItem(item: ApiItem) {
-      item.name = item.title.toLowerCase();
-      // convert 'secure' property to boolean `securityRisk`
-      item.securityRisk =  item.secure !== 'false';
-      // 'let' and 'var' doc types should be treated as 'const'
-      const docType = item.docType;
-      if (docType === 'let' || docType === 'var') {
-        item.docType = 'const';
-      }
     }
```
Comment (gkalpak) [inline aio/src/app/embedded/api/api.service.ts]: This should be rmoved as well. This is also why Travis fails and we get no preview 😞

Comment (petebacondarwin) [inline aio/src/app/embedded/api/api.service.ts]: Doh!

**Review by gkalpak — APPROVED** [review]
LGTM

**Review by gkalpak — APPROVED** [review]

**Pull request merged**